src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1

```diff
@@ -0,0 +1,4 @@
+[CmdletBinding()]
+param()
+
+Add-CapabilityFromApplication -Name 'AWS_Cli' -ApplicationName 'aws'
```

Comment:
You can take reference of npm, gulp, and other node tool capability. basic you will have the .ps1 for windows, and modify /src/Agent.Listener/Capabilities/NixCapabilitiesProvider.cs for linux and osx.  src/Misc/layoutbin/powershell/Add-NodeToolCapabilities.ps1